osu.Game/Overlays/Wiki/WikiHeader.cs

```diff
@@ -53,7 +52,7 @@ private void onWikiPageChange(ValueChangedEvent<APIWikiPage> e)
 
         private void onCurrentChange(ValueChangedEvent<string> e)
         {
-            if (e.NewValue == TabControl.Items.LastOrDefault())
+            if (e.NewValue == TabControl.Items.LastOrDefault() && WikiPageData.Value != null)
```

Comment:
This is really weird to read. I'd add an inline comment at very least explaining what each of these is intending to block.

Reply:
I still don't get it with the updated comment. What does "blank" entail here?

Reply:
I *think* that the "blank" is supposed to be matching the change in `WikiOverlay` (ie. if a wiki page request fails, the bindable's value is set to `null`).

I'm not sure this is a great way to fix this issue in general UX-wise, since the user will still be stuck at a blank screen and will have to essentially guess to click the index breadcrumb. The code checking this is also weird (why is the last item checked instead of the index item?)

I think I'd rather show a placeholder where the wiki content is supposed to go showing a short error message and a button/link that redirects back to index/previous page.

Reply:
Yeah, after I think about it again, the best thing to do is adding placeholder instead of blank page.

I don't like approach in this PR either.